packages/circularprogress/src/index.d.ts

```diff
@@ -1,24 +1,39 @@
-import { ForwardRefExoticComponent, RefAttributes } from 'react'
+import { ForwardRefExoticComponent, HTMLAttributes, RefAttributes } from 'react'
 
-interface Styling {
-  css?: Record<string, string | number>
```

Comment:
`css` isn't needed anymore. this is something we should have removed when we dropped glamourous